.github/secrel/config.yml

```diff
@@ -37,25 +40,31 @@ images:
 - name: vro-console
   context: "./console/build/docker"
   path: "./console/src/docker/Dockerfile"
+  args:
+    - JAR_FILE=console-*.jar
+    - ENTRYPOINT_FILE=entrypoint.sh
 - name: vro-service-data-access
   context: "./svc-lighthouse-api/build/docker"
   path: "./svc-lighthouse-api/src/docker/Dockerfile"
+  args:
+    - JAR_FILE=svc-lighthouse-api-*.jar
+    - ENTRYPOINT_FILE=entrypoint.sh
```

Comment:
These docker `build-args` need to be set for SecRel to build the Java-based images correctly.